modules/apps/collaboration/blogs/blogs-test/src/testIntegration/java/com/liferay/blogs/service/impl/test/BlogsEntryLocalServiceImplTest.java

```diff
@@ -108,7 +116,40 @@ public void testDeleteDiscussion() throws Exception {
 				BlogsEntry.class.getName(), blogsEntry.getEntryId()));
 	}
 
+	@Test
+	public void testUpdateStatusWithNullThemeDisplay() throws Exception {
+		ServiceContext serviceContext =
+			ServiceContextTestUtil.getServiceContext();
+
+		BlogsEntry blogsEntry = BlogsEntryLocalServiceUtil.addEntry(
+			TestPropsValues.getUserId(), StringUtil.randomString(),
+			StringUtil.randomString(), new Date(), serviceContext);
+
+		_blogsEntries.add(blogsEntry);
+
+		blogsEntry.setStatus(WorkflowConstants.STATUS_PENDING);
```

Comment:
hey @ChrisKian I don't see the purpose of setting the status to the entry if we don't persist it in the database. 
Instead of doing this, could we add a blog entry as if it had workflow so it states in DRAFT status? Maybe we can just invoke the logic of https://github.com/liferay/liferay-portal/blob/master/modules/apps/collaboration/blogs/blogs-test-util/src/main/java/com/liferay/blogs/util/test/BlogsTestUtil.java#L42 ?